packages/client/src/forms/configuration/default.ts

```diff
@@ -4332,6 +4337,1326 @@ export const registerForms: IDefaultRegisterForms = {
+                initialValue: '',
+                validate: [],
+                placeholder: formMessageDescriptors.formSelectPlaceholder,
+                options: [
+                  {
+                    value: 'MARRIAGE_USUAL_RESIDENCE',
+                    label: formMessageDescriptors.placeOfDeathSameAsPrimary
+                  },
+                  {
+                    value: 'OTHER',
+                    label: formMessageDescriptors.otherInstitution
+                  }
+                ],
+                mapping: {
+                  mutation: {
+                    operation: 'marriageEventLocationMutationTransformer',
+                    parameters: []
+                  },
+                  query: {
+                    operation: 'eventLocationTypeQueryTransformer',
+                    parameters: []
+                  }
+                }
+              },
+              {
+                name: 'typeOfMarriage',
+                type: 'SELECT_WITH_OPTIONS',
+                label: formMessageDescriptors.typeOfMarriage,
+                required: false,
+                initialValue: '',
+                validate: [],
+                placeholder: formMessageDescriptors.formSelectPlaceholder,
+                options: [
+                  {
+                    value: 'MONOGAMY',
+                    label: formMessageDescriptors.monogamy
+                  },
+                  {
+                    value: 'POLYGAMY',
+                    label: formMessageDescriptors.polygamy
+                  }
+                ],
+                mapping: {
+                  mutation: {
+                    operation: 'sectionFieldToBundleFieldTransformer',
+                    parameters: ['typeOfMarriage']
+                  },
+                  query: {
+                    operation: 'bundleFieldToSectionFieldTransformer',
+                    parameters: ['typeOfMarriage']
+                  },
+                  template: {
+                    fieldName: 'typeOfMarriage',
+                    operation: 'selectTransformer'
+                  }
+                }
+              }
+            ]
+          }
+        ]
+      },
+      {
+        id: MarriageSection.WitnessOne,
+        viewType: 'form',
+        name: formMessageDescriptors.witnessName,
+        title: formMessageDescriptors.witnessTitle,
+        groups: [
+          {
+            id: 'witness-view-group',
+            fields: [
+              {
+                name: 'seperator',
+                type: 'SUBSECTION',
+                label: {
+                  defaultMessage: 'Witness 1',
+                  description: 'Witness 1 desc',
+                  id: 'form.field.label.witness1'
+                },
+                initialValue: '',
+                ignoreBottomMargin: true,
+                validate: [],
+                conditionals: []
+              },
+              {
+                name: 'witnessOneFirstNamesEng',
```

Comment:
It might make things easier if we don't have "witnessOne" in the name of the fields

Reply:
yes. Changed it.